.github/workflows/gh-pages-deploy.yml

```diff
@@ -15,14 +15,14 @@ jobs:
         ]
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-java@v1
-        with:
-          java-version: 17
+      # https://cashapp.github.io/hermit/usage/ci/
+      - name: Init Hermit
+        uses: cashapp/activate-hermit@v1
       - uses: gradle/gradle-build-action@v2
```

Comment:
I think for the Gradle step you'll need to tell it where the Hermit Gradle is. IIRC there's a parameter to configure it.

Reply:
Relating to the caching, or the action itself? I'm not seeing the option here: https://github.com/gradle/actions/tree/main/setup-gradle

cc @leordev who I think set up the action originally

Reply:
I thought it existed, but I can't see any option for it :(

You might be better off disabling gradle/gradle-build-action, as it will always (AFAICT) download new versions of Gradle.